infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -58,6 +58,46 @@ xrt_versioned_builds = [
 # Built on push to specific tag.
 versioned_builds = [
   # Remove libtpu from PyPI builds
+  {
+    git_tag         = "v2.2.0"
+    pytorch_git_rev = "v2.2.0"
```

Comment:
FYI, PyTorch doesn't apply the `v2.2.0` tag until the final release. They'll have `v2.2.0rc0`, `v2.2.0rc1`, etc. I actually suggest we do the same for this release rather than updating the `v2.2.0` tag multiple times.

Reply:
Thanks, can you elaborate more about the solution? Should we add `v2.2.0rc0`, `v2.2.0rc1` build etc. here in advance?

Reply:
Thanks for this, Will

Hi, @zpcore, FYI, since we want to add trigger for 2.2.0 wheel to test in xl-ml-dashboard, and we won't catch `pytorch_git_rev` tag `v2.2.0` until PyTorch 2.2 release from PyTorch, and we also won't catch any other tag like `v2.2.0rc0`/`v2.2.0rc1` until PyTorch 2.2.0 branch cut date, we could merge this PR now but the trigger could build wheel successfully until they catch the tag

So for wheel on xl-ml-dashboard, let's use nightly wheel of torch and torch_xla until branch cut date

cc @lsy323

Reply:
> Thanks, can you elaborate more about the solution? Should we add `v2.2.0rc0`, `v2.2.0rc1` build etc. here in advance?

let's modify `git_tag` and `pytorch_git_rev` to `"v2.2.0rc0"` in this PR, we would modify again once PyTorch has other wheels built

cc @will-cromar, would you mind help to confirm it?

Reply:
Thanks, we have removed `pytorch_git_rev`.